docs/gallery/interactive/sofar_introduction.ipynb

```diff
@@ -0,0 +1,369 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This will check specific rules determined by the SOFA standard AES69 and general rules such as:\n",
+    "\n",
+    "- Are all mandatory data contained?\n",
+    "- Are the names of variables and attributes in accordance with the SOFA\n",
+    "  standard?\n",
+    "- Are the data types in accordance with the SOFA standard?\n",
+    "- Are the dimensions of the variables consistent and in accordance\n",
+    "  to the SOFA standard?\n",
+    "- Are the values of attributes consistent and in accordance to the\n",
+    "  SOFA standard?\n",
+    "\n",
+    "If any violations are detected, an error is raised.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "## Reading and writing SOFA objects\n",
+    "\n",
+    "Note that you usually do not need to call ``sofa.verify()`` separately because it is by default called if you create, write, or read a SOFA object. To write your SOFA object to disk type"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sf.write_sofa(os.path.join('..', '..', 'resources', 'my_first.sofa'), sofa)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "It is good to know that SOFA files are essentially netCDF4 files which is\n",
+    "based on HDF5. They can thus be viewed with [HDF View](https://www.hdfgroup.org/downloads/hdfview/).\n",
+    "\n",
+    "To read your sofa file you can use"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sofa_read = sf.read_sofa(os.path.join(\n",
+    "    '..', '..', 'resources', 'my_first.sofa'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And to see that the written and read files contain the same data you can check"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sf.equals(sofa, sofa_read)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Upgrading SOFA files\n",
+    "\n",
+    "SOFA conventions might get updates to fix bugs in the conventions, in case new conventions are introduced, or in case conventions get deprecated. To find out if SOFA file is up to date, call `upgrade_convention` to get a list of upgrade choices or the information that the data is up to date."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sofa.upgrade_convention()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Next steps\n",
+    "\n",
+    "For detailed information about sofar and more examples please refer to the [sofar documentation](https://sofar.readthedocs.io)."
```

Comment:
maybe add here the notebook link to working with pyfar and sofar

Reply:
Yes, added a place holder link to the pyfar introduction for later